backend/compact-connect/lambdas/cloudfront-csp/README.md

```diff
@@ -0,0 +1,54 @@
+# Cloudfront Viewer Response Edge Lambda
```

Comment:
One remaining item- 

Seems like the unsafe-eval is needed for i18n? I get an error on deploy that says:

```EvalError: Refused to evaluate a string as JavaScript because 'unsafe-eval' is not an allowed source of script in the following Content Security Policy directive: "script-src 'self'".```

Reply:
Oh, interesting. I'm not seeing that in our development environment. Are you sure you're deploying the frontend with the latest build from `yarn build`?

Reply:
Honestly, that is a fair question. I'll be back...

Reply:
@jsandoval81 Forgot I needed to upload the `dist/` folder to the S3 bucket.

Regardless, the UI loads, but I get the below error:

```
Refused to connect to 'https://7suc1ey74m.execute-api.us-east-1.amazonaws.com/joshua/mock/providers/query' because it violates the following Content Security Policy directive: "connect-src 'self' https://api.compactconnect.org  https://cognito-idp.us-east-1.amazonaws.com".
```

Is this a configuration issue on my end, or is this a case we didn't handle?

Reply:
Technically, a config issue in your stack. The CSP is working as expected in this case by refusing to let the frontend connect to your personal server-side (`https://7suc1ey74m.execute-api.us-east-1.amazonaws.com`).

If that's ok for now then it looks like we're good. Otherwise, we'll need to figure out the best way to add your personal environment to the CSP. That would be most straightforward if you had a long-lived domain for the both your frontend & backend.

Reply:
I don't think we decided on a hard line or anything, but I believe we decided that we would handle two cases for UI:
1) Create a domain name and we support the hosted UI infrastructure (and the CSP works)
2) Run the UI from localhost, point to the API in the cloud (CSP is irrelevant)

Reply:
Hmm OK. This is important though not an immediate priority but, eventually, it should be possible to stand up a test site in a few minutes with at most a few commands and basic technical knowledge. I'm concerned that this is another barrier to entry for doing so.

I plan on sharing the steps I had to take to stand up the site, which I may convert into a PR on the repo's README for your review. Let's continue this discussion at that time and, for now, operate under the status quo described by @jusdino (create a domain name, or run from localhost).